platform/podjs/src/browserPod.ts

```diff
@@ -483,10 +572,21 @@ function createUUID(): string {
 }
 
 let files = new Map<string, Stats>();
+
+/**
+ * The BrowserPolyNav @class implements the PolyNav interface.
+ */
 class BrowserPolyNav implements PolyNav {
     static readonly filesKey = "files";
+    /**
+     * Creating a new object with a property called actions. The actions property is an
+     * object with a string key and a value of a function that returns void.
+     */
```

Comment:
Well..... see my comment above :D Why duplicate the code with _more_ words? Could be incredibly confusing if the code changes and it's not addressed in the comment. OTOH, I don't think it adds _any_ value.